rasa/utils/tensorflow/models.py

```diff
@@ -798,11 +801,34 @@ def _get_mask_for(
     def _get_sequence_feature_lengths(
         self, tf_batch_data: Dict[Text, Dict[Text, List[tf.Tensor]]], key: Text
     ) -> tf.Tensor:
-        """Fetches the sequence lengths of real tokens per input example.
+        """Tries to create sequence length information for the given key.
+
+        Assumes that the batch data contains sequence-level features for the given
+        key, tries to fetch the sequence length information for that key from the given
+        batch, and turns it into a tensor.
+
+        If no sequence length information is present, it assumes the respective
+        sequence-level features are all empty and hence returns a tensor containing
+        0s only. This is useful in case we deal with empty sequences e.g. when
```

Comment:
Consider earlier comments.

Reply:
I can remove this paragraph now. Had eventually decided to just raise an error in that case. Otherwise, I'd still think this docstr would be useful because I wouldn't know what to expect from this when there are no sequence features.

Reply:
changed and simplified this again - unsurprisingly, there's not really a need to give create_model_data and create_label_data different "configurations" (because create_model_data kind of includes label data processing)